thoth/storages/graph/postgres.py

```diff
@@ -4133,7 +4133,8 @@ def _system_symbols_analysis_result(
         self,
         package_extract_run: PackageExtractRun,
         document: dict,
-        software_environment: SoftwareEnvironment or ExternalSoftwareEnvironment
+        software_environment: SoftwareEnvironment or ExternalSoftwareEnvironment,
```

Comment:
```python
        software_environment: Union[SoftwareEnvironment, ExternalSoftwareEnvironment],
```

Reply:
Thanks :) I changed also in the other functions :beers: